### DISCUSSION TITLE
Conflicts between default Windows 11 Light and Dark themes on multi-virtual desktop setup

### DISCUSSION THREAD

**Discussion by JohnRenzFlores** [AutoDarkMode/Windows-Auto-Night-Mode#519] [Q&A]
Why is it that both the default Windows 11 light and dark theme is not automatically applied all throughout the virtual desktop? Only the virtual desktop where the Auto Dark Mode X is run, and on other virtual desktop is not changing. When using different themes, this issue is not evident.

**Comment by Spiritreader** [comment]
I can't really answer these questions unless you fill out a full bug report card, include windows version, logs, precise auto dark Mode version and Screenshots / steps how how to reproduce the issue.

Without that it's a bit like asking why the sun is green.
